pom.xml

```diff
@@ -29,12 +29,14 @@
   <description>Testing tools for Apache Accumulo</description>
   <properties>
     <accumulo.version>2.1.0-SNAPSHOT</accumulo.version>
+    <disruptor.version>3.4.4</disruptor.version>
     <eclipseFormatterStyle>${project.basedir}/contrib/Eclipse-Accumulo-Codestyle.xml</eclipseFormatterStyle>
     <hadoop.version>3.2.0</hadoop.version>
+    <log4j2.version>2.14.1</log4j2.version>
```

Comment:
Not sure if it's useful to have a property for this, if you use the BOM, since the BOM should eliminate the need to use this property more than one time. However, we may still want to be able to set the value with `-Dlog4j2.version=` when building to override this, in which case, the property would still need to be here, even if it's only ever used once (for the BOM).